src/Components/WebAssembly/testassets/Wasm.Authentication.Server/Startup.cs

```diff
@@ -37,6 +37,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddAuthentication()
                 .AddIdentityServerJwt();
 
+            services.AddWebAssemblyStaticFilesConfiguration();
```

Comment:
I suspect we'll end up changing this method name further, since it's a little odd to conflate "Blazor WebAssembly" with "WebAssembly in general".

However unless you're keen to address this right away, I recommend leaving this until the API review.

Reply:
It's not going to be there in a couple of hours.